target/linux/ath79/image/generic.mk

```diff
@@ -154,6 +154,23 @@ define Device/buffalo_wzr-hp-ag300h
 endef
 TARGET_DEVICES += buffalo_wzr-hp-ag300h
 
+define Device/buffalo_wzr-hp-g300nh
+  ATH_SOC := ar9132
+  DEVICE_TITLE := Buffalo WZR-HP-G300NH
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ledtrig-usbport kmod-gpio-nxp-74hc153
+  BLOCKSIZE := 128k
+  IMAGE_SIZE := 32128k
+  IMAGES += factory.bin tftp.bin
+  IMAGE/default := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | \
+  pad-rootfs | check-size $$$$(IMAGE_SIZE)
```

Comment:
Indent for wrapped line should be one tab. Despite, check-size does not need an argument in recent master.